src/Umbraco.Web.UI.NetCore/Umbraco.Web.UI.NetCore.csproj

```diff
@@ -4,35 +4,52 @@
         <TargetFramework>netcoreapp3.1</TargetFramework>
         <RootNamespace>Umbraco.Web.UI.NetCore</RootNamespace>
     </PropertyGroup>
-
     <ItemGroup>
+      <ProjectReference Include="..\Umbraco.Examine.Lucene\Umbraco.Examine.Lucene.csproj" />
+      <ProjectReference Include="..\Umbraco.PublishedCache.NuCache\Umbraco.PublishedCache.NuCache.csproj" />
       <ProjectReference Include="..\Umbraco.Web.BackOffice\Umbraco.Web.BackOffice.csproj" />
       <ProjectReference Include="..\Umbraco.Web.Common\Umbraco.Web.Common.csproj" />
       <ProjectReference Include="..\Umbraco.Web.Website\Umbraco.Web.Website.csproj" />
     </ItemGroup>
 
     <ItemGroup>
       <Folder Include="wwwroot\Media" />
+      <Folder Include="wwwroot\Umbraco\views\install" />
     </ItemGroup>
 
     <ItemGroup>
-      <Compile Remove="wwwroot\Umbraco\**" />
         <Compile Remove="App_Data\**" />
     </ItemGroup>
 
     <ItemGroup>
-      <EmbeddedResource Remove="wwwroot\Umbraco\**" />
         <EmbeddedResource Remove="App_Data\**" />
     </ItemGroup>
 
     <ItemGroup>
-      <None Remove="wwwroot\Umbraco\**" />
         <None Remove="App_Data\**" />
     </ItemGroup>
 
     <ItemGroup>
-        <Content Remove="wwwroot\Umbraco\**" />
         <Content Remove="App_Data\**" />
+        <Content Remove="wwwroot\Umbraco\views\install\machinekey.html" />
```

Comment:
We can remove the machinekey install step since that doesn't actually exist in netcore